Markdown/cmm-2.0.14/system-operators/2-installation.md

```diff
@@ -810,10 +812,10 @@ To install a Log Agent, proceed as follows:
 2. Prepare an installation directory.
 3. Extract the CMM_client_2.0.14-x.tar.gz archive file from the CMM installation package to the
    installation directory. The archive provides the following files:
-     - `log-agent-CMM_2.0.x.run`
-     - `monasca-agent-CMM_2.0.x.run`
-     - `monasca-ui-1.17.x-CMM_2.0.x.tar.gz`
-4. Change the access permission of the `log-agent-CMM_2.0.x.run` file to Execute.
+     - `log-agent-7.3.0_2.0.1-CMM2.0.14-x.run`
+     - `monasca-agent-3.0.3-CMM2.0.14-x.run`
+     - `monasca-ui-1.17.2-CMM2.0.14-x.tar.gz`
+4. Change the access permission of the `log-agent-7.3.0_2.0.1-CMM2.0.14-x.run` file to Execute.
```

Comment:
Thanks for the update, \
Here I would rather leave the versions like:

- `log-agent-7.3.0_2.0.x-CMM2.0.14-x.run`  
- `monasca-agent-3.0.x-CMM2.0.14-x.run` 
- `monasca-ui-1.17.x-CMM2.0.14-x.tar.gz`

Because it is possible that we need to increment the version of the core libraries.